extensionsGallery-insider.json

```diff
@@ -4324,7 +4385,7 @@
 					"metadataItems": [
 						{
 							"name": "TotalCount",
-							"count": 71
+							"count": 72
```

Comment:
I'm not sure how this is going to be used. do we need to update this number if we use new `hidden` flag?

Seems like there is currently 69 extensions, but this was already stating 71, so I'm not sure if this affects anything at all.

![image](https://user-images.githubusercontent.com/11728682/135349354-535990bc-6218-4766-9234-41deb85cd44d.png)

Reply:
Yeah you should still update it.